lib/aff4.py

```diff
@@ -392,6 +392,10 @@ def _UpdateChildIndex(self, urn, token):
 
         try:
           self.intermediate_cache.Get(urn)
+          # When an object is retrieved from the cache bump it back to the
+          # front to ensure heavily accessed items remain in the cache for their
+          # full age.
+          self.intermediate_cache.Put(urn, 1)
```

Comment:
Yeah, I just reread through the Gets and saw how it is already moving it to the front of the list in FastStore.  Once every 10 minutes should be fine, but when we are getting the collisions we are seeing it happen way more often.  I figured it was just getting pushed out due to lots of new aff4 objects being created when processing bulk collection file/registry results.